api/live.js

```diff
@@ -105,6 +110,34 @@ async function handleContrib(userId, req, res) {
   }
 }
 
+async function getMultiplierForTask(taskId) {
```

Comment:
MANY TO MANY!

task.hasMany(Multipliers, through: { grouping} )

Reply:
otherwise reaplace findOne where: id to findByPk(id)

Reply:
Yeah, I've just been so conditioned to not think in M:N relations because SQL, so it's doomed for me. Curse having competing mental models and having the ORM support both.